source/triggers/aws-eventbridge.txt

```diff
@@ -152,6 +164,238 @@ Procedure
+                  console.log(`DB: ${changeEvent.ns.db}`);
+                  console.log(`Collection: ${changeEvent.ns.coll}`);
+                  console.log(`Operation type: ${changeEvent.operationType}`);
+
+                  // Throw an error in your function to suspend the trigger and stop processing additional events
+                  throw new Error(`Encountered error: ${error.message}`);
+               };
+
+         .. step:: Add the Function to the ``config.json``
+
+            Add the function's configuration to the ``Function`` folder's 
+            ``config.json`` file. The file should resemble the following:
+
+            .. code-block:: json
+
+               [
+                  {
+                     "name": "<Error Handler Function Name>",
+                     "private": <boolean>
+                  }
+               ]
+
+            For more information on function configuration files, 
+            see :ref:`Function Configuration Files <appconfig-functions>`.
+
+         .. step:: Add an Error Handler to Your Trigger Configuration
+
+            Add an ``error_handler`` attribute to your trigger configuration file
+            in the ``Triggers`` folder. The trigger configuration file should 
+            resemble the following:
+
+            .. code-block:: json
+
+                     {
+                     "name": "...",
+                     "type": "...",
+                     "event_processors": {
+                        "AWS_EVENTBRIDGE": {
+                              "config": {
+                                 "account_id": "<AWS Account ID>",
+                                 "region": "<AWS Region>",
+                                 "extended_json_enabled": <boolean>
+                              }
+                           }
+                        },
+                        "error_handler": {
+                           "config": {
+                              "enabled": <boolean>,
+                              "function_name": "<Error Handler Function Name>"
+                           }
+                        }
+                     }
+            
+            For more information on trigger configuration files, see
+            :ref:`Trigger Configuration Files <appconfig-triggers>`.
+
+.. _eventbridge-error-handler-parameters:
+
+Error Handler Parameters
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The default error handler has two parameters: ``error`` and ``changeEvent``.
+
+``error`` 
+`````````
+Has two attributes: ``code`` and ``message``.
+
+- ``code``: For a list of error codes 
+used by the error handler, see the below section on :ref:`Error 
+Codes <eventbridge-error-codes>`. 
+
+- ``message``: The unfiltered error message from an errored Eventbridge
+put request.
+
+``changeEvent``
+```````````````
+
+The requested change to your data made by EventBridge. For more information
+on types of change events and their configurations, see 
```

Comment:
would it be more helpful to link to the Database Triggers page's [Change Event Types section](https://www.mongodb.com/docs/atlas/app-services/triggers/database-triggers/#change-event-types)? or are they different for EventBridge events?